README.md

```diff
@@ -27,6 +27,12 @@ To run one of the benchmarks:
 
 4. For good measure, execute `$ bash ../clean.sh` when finished.
 
+### Interpretation
+
+The numbers represent the real execution time (wall-clock time) it took for each language to execute the given task. **A lower number indicates better performance.**
+
+`bash ../run.sh` runs each program three times using the `runOnce` function and `awk` captures the real execution time; however, only the third run is recorded and outputted.
```

Comment:
https://github.com/bddicken/languages/blob/7d2d3ee6691335456a499975cfb2b1e9fe4937da/run.sh#L7-L12

this is incorrect, all three runs are recorded and outputted.

Reply:
Oops, I just noticed this as well. I got a little jumbled up as i was reading the readme because it was only showing one number. I'll fix this right now.